Libraries/Text/TextInput/Multiline/RCTMultilineTextInputViewManager.m

```diff
@@ -17,8 +19,29 @@ - (UIView *)view
   return [[RCTMultilineTextInputView alloc] initWithBridge:self.bridge];
 }
 
+- (RCTShadowView *)shadowView
+{
+  RCTBaseTextInputShadowView *shadowView = (RCTBaseTextInputShadowView *)[super shadowView];
+ 
+  shadowView.maximumNumberOfLines = 0;
+  shadowView.exactNumberOfLines = 0;
+ 
+  return shadowView;
+}
+
 #pragma mark - Multiline <TextInput> (aka TextView) specific properties
 
 RCT_REMAP_VIEW_PROPERTY(dataDetectorTypes, backedTextInputView.dataDetectorTypes, UIDataDetectorTypes)
 
+RCT_CUSTOM_SHADOW_PROPERTY(maximumNumberOfLines, NSNumber, RCTBaseTextInputShadowView)
```

Comment:
I think you can use something like this instead https://github.com/facebook/react-native/blob/11570e71a2747602ff485552094b413375b19a96/Libraries/Text/Text/RCTTextViewManager.m#L29

Reply:
Probably dont need the remap version for the first one since it has same name